cosmos-gui/README.md

```diff
@@ -238,6 +241,83 @@ Next coming versions of the GUI will allow the users to explore their HDFS space
 
 [Top](#top)
 
+##<a name="administration"></a>Administration
+
+Two are the sources of data, the logs and the database, useful for an admistrator of cosmos-gui.
+
+[Top](#top)
+
+###<a name="loggingtraces"></a>Logging traces
+Logging traces, typically saved under `/var/log/cosmos/cosmos-gui`, are the main source of information regarding the GUI performance. These traces are written in JSON format, having the following fields: level, message and timestamp. For instance:
+
+    {"level":"info","message":"cosmos-gui running at http://localhost:9090","timestamp":"2015-07-23T13:25:20.019Z"}
+
+Logging levels follow this hierarchy:
+
+    debug < info < warn < error < fatal
+    
+Within the log it is expected to find many `info` messages, and a few of `warn` or `error` types. Of special interest are the errors:
+
+* ***There was some error when connecting to MySQL database. The server will not be run***: This message may appear when starting the GUI. Most probably the MySQL endpoint is not correct, the MySQL user is not allowed to remotely connect, of there is some network error like a port filtering.
+* ***There was some error when getting user information from the database***: This message may appear when a user gets the main page and his/her session has not yet expired; then, his/her information is retrieved from the database. Most probably some network error is avoiding to get that information, since the initial connection to the database was successful. 
+* ***There was some error when getting user information from the IdM***: This message may appear when a user signs in using his/her Identity Manager (IdM) credentials. Most probably the IdM endpoint is not correct, the client id and secret related to cosmos-gui are not correct or the callback URL has not been propertly set.
+* ***There was an error while setting up the password for user XXX***: This message may appear when a user from the old Cosmos deployment has accessed the GUI for the first time. Most probably some network error is avoiding to get that information, since the initial connection to the database was successful.
```

Comment:
In general I think is more smart to use `for user <user>` or something similar instead of using `XXX`, but it's a matter of style (feel free of marking this comment as NTC).